pystac/item.py

```diff
@@ -247,3 +256,9 @@ def from_dict(d):
                      media_type=media_type,
                      title=title,
                      properties=properties)
+
+    def is_eo(self):
+        if self.properties:
+            if self.properties.get('eo:bands', None):
+                return True
+        return False
```

Comment:
I don't think core classes should reference extensions. Is there a way to do whatever this is needed for on the EOItem class instead? Can the user just check the type instead of using this?

Reply:
Ok, that makes sense. they couldn't check the type because I use this to iterate over the assets within and item and check whether they should be converted to EO assets. I just turned it into a standalone function within eo.py. I could also put it in utils if that made more sense.